synapse/res/templates/recaptcha.html

```diff
@@ -9,7 +9,7 @@
 <link rel="stylesheet" href="/_matrix/static/client/register/style.css">
 <script>
 function captchaDone() {
-    $('#registrationForm').submit();
```

Comment:
can we not remove the `<script src="//code.jquery.com/jquery-1.11.2.min.js">` line?

Reply:
> can we not remove the `<script src="//code.jquery.com/jquery-1.11.2.min.js">` line?

I'm not sure about it, I'll look into it and try it out :)

Reply:
>can we not remove the <script src="//code.jquery.com/jquery-1.11.2.min.js"> line?

Done:), can you verify it?